src/mono/browser/runtime/types/index.ts

```diff
@@ -203,6 +203,7 @@ export type ResourceExtensions = { [extensionName: string]: ResourceList };
 
 export interface ResourceGroups {
     hash?: string;
+    fingerprinting?: { [name: string]: string },
```

Comment:
please consider impact on `getCacheKey` in `src\mono\browser\runtime\interp-pgo.ts`

Reply:
The whole resources object gets removed `delete inputs.resources;`